configure.py

```diff
@@ -1496,17 +1503,43 @@ def _generate_target_source_files_isa_properties(fd, target):
     @staticmethod
     def _write_header(fd):
         fd.write('cmake_minimum_required(VERSION 2.8.0)\n')
-        fd.write('project(botan LANGUAGES CXX)\n\n')
+        fd.write('project(botan)\n\n')
+        fd.write('if(POLICY CMP0042)\n\n')
```

Comment:
"\n\n" -> "\n" for the three lines that belong together without empty lines in between